docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -229,6 +395,24 @@ Clears the _context_, and returns `null`.
 
 Includes or excludes columns from a data table. If you specify both, this will exclude first.
 
+*Expression syntax*
+[source,js]
+----
+columns include=”@timestamp, projects, cost”
+columns exclude=”username, country, age”
+----
+
+*Code example*
+[source,text]
+----
+filters
+| demodata
+| columns include="price, cost, state, project"
+| table
+| render
+----
+This only keeps the `price`, `cost`, `state`, and `project` columns from the demodata datasource and removes all other columns.
```

Comment:
`demodata datasource` > `demo data source`

Reply:
demodata is the name of the function so instead what if I make it > `demodata` data source